bin/packages/build-types.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable eslint-comments/disable-enable-pair, no-console */
```

Comment:
I was under the impression we had some rule in place to enforce that all `eslint-disable` are complimented by an `eslint-enable`, even for these sorts of cases where we intend for the whole file to be disabled.

e.g.

https://github.com/WordPress/gutenberg/blob/34ed2a6042d42fa18a5dcd0853d59bdff6a068d9/bin/packages/build.js#L1

https://github.com/WordPress/gutenberg/blob/34ed2a6042d42fa18a5dcd0853d59bdff6a068d9/bin/packages/build.js#L184

Maybe some issue when disabling multiple in one go? Not really an issue for this pull request specifically, just wondering if it's something we should be aware of.

Reply:
> I was under the impression we had some rule in place to enforce that all `eslint-disable` are complimented by an `eslint-enable`, even for these sorts of cases where we intend for the whole file to be disabled.
> Maybe some issue when disabling multiple in one go?

The other rule disabled in the multiple disable is what disabled the lint rule enforcing dis/enable pairs 😂 `eslint-comments/disable-enable-pair`.

🤔 I could add the enable at the end of the file to ensure no other less desirable dis/enable pairs sneak into this file. It might be worth overriding the no-console rule in this directory as well.